lib/net/peer.js

```diff
@@ -1900,6 +1903,43 @@ class Peer extends EventEmitter {
     return false;
   }
 
+  /**
+   * Increase dosscore on peer.
+   * @param {Number} dosScore
+   * @param {Number} time
+   * @returns {Boolean}
+   */
+
+  increaseDos(dosScore, time) {
+    // dosScore is a time-bound transient banscore useful to detect DOS
+    // attacks by repeated requests for a message (e.g. getproof).  Calling
+    // increaseBan with a dosScore multiple times will increase dosScore in a
+    // logarithmically decaying fashion wrt time between requests to penalize
+    // short bursts of repeated requests.  The dosScore decays back to 0 in
+    // 30mins, i.e. 1800s.
+    const seconds = (time - this.lastDos) / 1000;
+    if (seconds > common.DOS_RESET_TIME) {
+      this.dosScore = 0;
+    } else if (this.dosScore > 1 && seconds > 0) {
+      if (seconds < 64) {
```

Comment:
Nit: no need for curly braces with 1 liners

Reply:
Fixed.